src/main/java/bio/terra/service/snapshot/SnapshotService.java

```diff
@@ -177,6 +200,7 @@ public String createSnapshot(
       SnapshotRequestModel snapshotRequestModel,
       Dataset dataset,
       AuthenticatedUserRequest userReq) {
+    snapshotRequestModel.setName(generateUpdatedSnapshotNameIfByRequest(snapshotRequestModel));
```

Comment:
Is there a length constraint on model names in TDR's db?

Reply:
I was seeing one up to 255 characters, I think. I think we check that elsewhere though.